pex/executor.py

```diff
@@ -56,47 +58,40 @@ def __init__(self, cmd, exc):
       self.exc = exc
 
   @classmethod
-  def open_process(cls, cmd, env=None, cwd=None, combined=False, **kwargs):
+  def open_process(cls, cmd, **kwargs):
```

Comment:
Why remove these two kwargs? Although they can off course still be passed through `**kwargs`, if they are used frequently I would argue explicit is better than implicit.